src/app/content/studyGuides/selectors.ts

```diff
@@ -81,21 +88,57 @@ export const loadedCountsPerSource = createSelector(
   getLoadedCountsPerSource
 );
 
+export const highlightColorFiltersWithContent = createSelector(
+  totalCountsPerPageOrEmpty,
+  getHighlightColorFiltersWithContent
+);
+
+export const defaultLocationFilter = createSelector(
+  studyGuidesLocationFilters,
+  parentSelectors.page,
+  (filters, currentPage) => currentPage && getHighlightLocationFilterForPage(filters, currentPage)
+);
+
+const filtersFromQuery = createSelector(
+  navigationSelectors.query,
+  (query) => getFiltersFromQuery(query)
+);
+
+export const unloggedAndQueryMissingFirstChapter = createSelector(
+  authSelectors.loggedOut,
+  parentSelectors.firstChapter,
+  filtersFromQuery,
+  (notLoggedIn, firstChapter, queryFilters) =>
+    notLoggedIn && firstChapter && !queryFilters.locationIds.includes(firstChapter.id)
```

Comment:
a logged out user can't change filters right?

Reply:
correct

Reply:
hmm ok, looks like they still can via the query. will adjust.